### PR TITLE
ProxylessNAS 번역

### PR DESCRIPTION
**Pull request by cpprhtn**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #59 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

ProxylessNAS 문서 번역

### CODE REVIEW

File: pytorch_vision_proxylessnas.md [hunk 1]
```diff
@@ -20,28 +20,27 @@ demo-model-link: https://huggingface.co/spaces/pytorch/ProxylessNAS
...
 model = torch.hub.load('mit-han-lab/ProxylessNAS', target_platform, pretrained=True)
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델은 동일한 방식으로 정규화된 입력 이미지를 요구합니다.
+i.e. `H`와 `W`가 최소 `224`의 크기를 가지는 `(3 x H x W)`형태의 3채널 RGB 이미지의 미니배치
```
Comment (yh00214) [inline pytorch_vision_proxylessnas.md:29]: 다른 대부분의 번역에서 `i.e`를 `즉,` 으로 번역하고, 자연스러운 문장 구조를 띄도록 변경하였습니다. 원문 문장 구조를 유지하기 위해 이렇게 번역하신 것 같은데, 다른 번역과 통일성을 생각하면 자연스러운 문장 구조로 번역하시는 것도 좋을 것 같습니다.

**Review by yh00214 — CHANGES_REQUESTED** [review]
번역 수고하셨습니다! 전체적으로 훌륭합니다~

File: pytorch_vision_proxylessnas.md (hunk 1 above)
Comment (cpprhtn) [reply pytorch_vision_proxylessnas.md:29]: 감사합니다! 수정해보겠습니다 :)

**Comment by KyubumShin** [comment]
전체적으로 잘 번역된 것 같습니다!

File: pytorch_vision_proxylessnas.md [hunk 2]
```diff
@@ -52,50 +51,50 @@ preprocess = transforms.Compose([
...
 ```
 
-### Model Description
+### 모델 설명
 
-ProxylessNAS models are from the [ProxylessNAS: Direct Neural Architecture Search on Target Task and Hardware](https://arxiv.org/abs/1812.00332) paper.
+ProxylessNAS 모델은 [ProxylessNAS: Direct Neural Architecture Search on Target Task and Hardware](https://arxiv.org/abs/1812.00332) 논문에서 제안되었습니다.
 
-Conventionally, people tend to design *one efficient model* for *all hardware platforms*. But different hardware has different properties, for example, CPU has higher frequency and GPU is better at parallization. Therefore, instead of generalizing, we need to **specialize** CNN architectures for different hardware platforms. As shown in below, with similar accuracy, specialization offers free yet significant performance boost on all three platforms.
+일반적으로, 사람들은 *모든 하드웨어 플랫폼*에 대해 *하나의 효율적인 모델*을 설계하는 경향이 있습니다. 하지만 하드웨어마다 특성이 다릅니다. 에를 들어 CPU는 더 높은 주파수를 가지지만 GPU는 병렬화에 더 뛰어납니다. 따라서 일반화하는 대신 다양한 하드웨어 플랫폼에 대한 CNN 아키텍처를 **전문화**해야 합니다. 아래에서 볼 수 있듯이, 전문화는 유사한 정확도를 가지는 결과에서 세 가지 플랫폼 모두에서 무료이지만 상당한 성능 향상을 제공합니다.
```
Comment (Woodywarhol9) [inline pytorch_vision_proxylessnas.md:89]: `모든 하드웨어 플랫폼(CPU, GPU, Mobile)`처럼 하드웨어 플랫폼이 뭔지 명시해주면 더 좋을 거 같아요!

`일반화하는 대신 하드웨어 플랫폼에 맞게 CNN 아키텍처를 **전문화**해야 합니다.`
문장을 이런 식으로 바꾸면 더 읽기 좋을 거 같아요!

`전문화는 세 가지 플랫폼 모두에서 상당한 성능 향상을 제공합니다.` 이렇게 해석하면 어떨까요?
`무료`라는 표현은 아마 논문에서 나오는 `search cost` 같은 것이 적다는  의미 같은데 별도의 표나 자료가 없이 갑자기 `무료`라는 표현이 나오면 이해하기 힘들다고 생각해서 제외했습니다.

**Review by Woodywarhol9 — COMMENTED** [review]
고생하셨습니다! 
몇 가지 수정 제안드려요

File: pytorch_vision_proxylessnas.md (hunk 2 above)
Comment (cpprhtn) [reply pytorch_vision_proxylessnas.md:89]: 모든 하드웨어 플랫폼에 대해서는 뒤에 문장에서 예시를 통해 언급해서 안적어도 괜찮을것 같습니다.

나머지 두가지 의견은 너무 좋은 문장인것 같습니다! 👍

File: pytorch_vision_proxylessnas.md [hunk 3]
```diff
@@ -52,50 +51,50 @@ preprocess = transforms.Compose([
...
-# move the input and model to GPU for speed if available
+# GPU 사용이 가능한 경우 속도를 위해 입력과 모델을 GPU로 이동
 if torch.cuda.is_available():
     input_batch = input_batch.to('cuda')
     model.to('cuda')
 
 with torch.no_grad():
     output = model(input_batch)
-# Tensor of shape 1000, with confidence scores over Imagenet's 1000 classes
+# Imagnet의 1000개 클래스에 대한 신뢰도 점수를 가진 1000 형태의 텐서 출력
```
Comment (codingbowoo) [inline pytorch_vision_proxylessnas.md:63]:
```suggestion
# Imagenet의 1000개 클래스에 대한 신뢰도 점수를 가진 1000 형태의 텐서 출력
```
오탈자가 있고, shape 1000을 좀 더 자연스럽게 번역할 수 있는 방법이 있을까 고민해보겠습니다.

File: pytorch_vision_proxylessnas.md [hunk 4]
```diff
@@ -52,50 +51,50 @@ preprocess = transforms.Compose([
...
 ```
 
-### Model Description
+### 모델 설명
 
-ProxylessNAS models are from the [ProxylessNAS: Direct Neural Architecture Search on Target Task and Hardware](https://arxiv.org/abs/1812.00332) paper.
+ProxylessNAS 모델은 [ProxylessNAS: Direct Neural Architecture Search on Target Task and Hardware](https://arxiv.org/abs/1812.00332) 논문에서 제안되었습니다.
 
-Conventionally, people tend to design *one efficient model* for *all hardware platforms*. But different hardware has different properties, for example, CPU has higher frequency and GPU is better at parallization. Therefore, instead of generalizing, we need to **specialize** CNN architectures for different hardware platforms. As shown in below, with similar accuracy, specialization offers free yet significant performance boost on all three platforms.
+일반적으로, 사람들은 *모든 하드웨어 플랫폼*에 대해 *하나의 효율적인 모델*을 설계하는 경향이 있습니다. 하지만 하드웨어마다 특성이 다릅니다. 에를 들어 CPU는 더 높은 주파수를 가지지만 GPU는 병렬화에 더 뛰어납니다. 따라서 일반화하는 대신 하드웨어 플랫폼에 맞게 CNN 아키텍처를 **전문화**해야 합니다. 아래에서 볼 수 있듯이, 전문화는 세 가지 플랫폼 모두에서 상당한 성능 향상을 제공합니다.
```
Comment (codingbowoo) [inline pytorch_vision_proxylessnas.md:89]:
```suggestion
일반적으로, 사람들은 *모든 하드웨어 플랫폼*에 대해 *하나의 효율적인 모델*을 설계하는 경향이 있습니다. 하지만 하드웨어마다 특성이 다릅니다. 예를 들어 CPU는 더 높은 주파수를 가지지만 GPU는 병렬화에 더 뛰어납니다. 따라서 모델을 일반화하기보다는 하드웨어 플랫폼에 맞게 CNN 아키텍처를 **전문화**해야 합니다. 아래에서 볼 수 있듯이, 전문화는 세 가지 플랫폼 모두에서 상당한 성능 향상을 제공합니다.
```
1. 오탈자가 있습니다. 
2. CPU가 더 높은 주파수를 가져서 어떤 점이 뛰어난지를 추가 설명해보는건 어떨까 하는 생각이 드네요.
3. specialize를 전문화 로 번역한게 좋네요!
4. 일부 읽기 좋게 문장을 변경해봤는데 어떠실까요?

**Review by codingbowoo — CHANGES_REQUESTED** [review]
같이 고민해볼 지점들이 있는 것 같습니다 :)

File: pytorch_vision_proxylessnas.md (hunk 3 above)
Comment (cpprhtn) [reply pytorch_vision_proxylessnas.md:63]: 텐서도 Tensor로 적기로 해서 그것도 고쳐야하네요 하하

File: pytorch_vision_proxylessnas.md (hunk 4 above)
Comment (cpprhtn) [reply pytorch_vision_proxylessnas.md:89]: 오탈자 넘 잘 찾으십니다!!
2번에대해서는 고민해봐야겠습니다
4번에 대한 의견은 너무 좋은것 같습니다 👍

**Review by codingbowoo — APPROVED** [review]
